lib/api/src/js/instance.rs

```diff
@@ -87,11 +84,11 @@ impl Instance {
     pub fn from_module_and_instance(
         mut store: &mut impl AsStoreMut,
         module: &Module,
+        externs: Vec<Extern>,
         instance: StoreHandle<WebAssembly::Instance>,
```

Comment:
That's another API change that should go in 3.0?

Reply:
not sure on this one as it will drag a bunch of other code with it - might be too much

Reply:
Does that mean the signature of the function can be rolled back to without the externs, and we forget about this change?